corgi/web/templates/rest_framework/api.html

```diff
@@ -0,0 +1,5 @@
+{% extends "rest_framework/base.html" %}
+
+{% block branding %}
+  <a class="navbar-brand" href="/">Component Registry</a>
+{% endblock %}
```

Comment:
Super minor, but browsers make lots of spammy requests for a favicon even when one doesn't exist / isn't defined on the page. If we're going to override the defaults, we may as well fix this issue too:
```suggestion
{% extends "rest_framework/base.html" %}

{% block style %}
{{ block.super }}
  <link rel="icon" href="data:," />
{% endblock %}

{% block branding %}
  <a class="navbar-brand" href="/">Component Registry</a>
{% endblock %}
```
Reference:
https://stackoverflow.com/questions/46842938/how-do-i-edit-the-favicon-in-the-browsable-api-in-django-rest-framework
https://stackoverflow.com/questions/1321878/how-to-prevent-favicon-ico-requests

Reply:
We don't have a favicon on the main page either so I guess I'd prefer to leave this as is and try and add one (that works in both places) in a separate PR.

Reply:
Ack, FYI you could also add the "empty favicon" snippet above to the index.html of the main page - anywhere in the \<head\> element will work. But leaving for the future and trying to use a real favicon is also fine.